clang/include/clang-c/Dependencies.h

```diff
@@ -380,6 +380,210 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v5(
     CXModuleLookupOutputCallback *MLO, unsigned Options,
     CXFileDependenciesList **Out, CXDiagnosticSet *OutDiags);
 
+/**
+ * Output of \c clang_experimental_DependencyScannerWorker_getDependenciesGraph.
+ */
+typedef struct CXOpaqueDependenciesGraph *CXDependenciesGraph;
+
+/**
+ * An individual module dependency that is part of an overall compilation
+ * \c CXDependenciesGraph.
+ */
+typedef struct CXOpaqueDepGraphModule *CXDepGraphModule;
```

Comment:
Why do we have short names in the types (DepGraph), but many of the functions below use the full name "_DependenciesGraph_"?

Reply:
"DependenciesGraph" in `clang_experimental_DependenciesGraph_getNumModules` refers to the object type (`CXDependenciesGraph`) excluding the `CX` prefix.
If the type changes to `CXDependencyGraph` the function name will be `clang_experimental_DependencyGraph_getNumModules`.

Reply:
I guess my question is why sometimes shorten it? If we're concerned about long names then probably we should shorten it everywhere. If not, long form everywhere.  I'm leaning toward short name -- while I expect libclang to be a bit verbose these are getting quite bad

Reply:
I'm fine with renaming `CXDependenciesGraph` to `CXDepGraph`; I mainly want to make it clear the function names should follow a certain convention, not rename them just to be shorter.

Reply:
Renamed `CXDependenciesGraph` -> `CXDepGraph` which affected the function names as well.